### PR TITLE
[ GRU ] Add GRU Unittest

### PR DESCRIPTION
**Pull request by jijoongmoon**
This commit includes,
  . unittests of gru layer
  . add keras code to compare

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: test/unittest/unittest_nntrainer_models.cpp [hunk 1]
```diff
@@ -1208,6 +1208,83 @@ INI multi_rnn_return_sequence_with_batch(
...
+    nn_base + "loss=mse | batch_size=2",
+    sgd_base + "learning_rate = 0.1",
+    I("input") + input_base + "input_shape=1:2:1",
+    I("gru") + gru_base +
+      "unit = 2" + "input_layers=input"+ "return_sequences=true",
+    I("outputlayer") + fc_base + "unit = 1" + "input_layers=gru"
+  }
+);
+
+INI gru_return_sequence_with_batch_n(
```
Comment (zhoonit) [inline test/unittest/unittest_nntrainer_models.cpp:1247]: As we have 10 iteration, could you check if this test is used?

**Review by zhoonit — APPROVED** [review]
LGTM with a minor comment

File: test/unittest/unittest_nntrainer_models.cpp (hunk 1 above)
Comment (jijoongmoon) [reply test/unittest/unittest_nntrainer_models.cpp]: this isn't used. deleted.

**Review by kparichay — APPROVED** [review]
LGTM

**Pull request merged**